src/lib/ruby-generator/control.js

```diff
@@ -4,12 +4,70 @@
  * @return {Blockly} Blockly defined Ruby generator.
  */
 export default function (Blockly) {
+    Blockly.Ruby.control_wait = function (block) {
+        const secs = Blockly.Ruby.valueToCode(block, 'DURATION', Blockly.Ruby.ORDER_NONE) || '0';
+        return `sleep(${secs})\n`;
+    };
+
     Blockly.Ruby.control_repeat = function (block) {
         const times = Blockly.Ruby.valueToCode(block, 'TIMES', Blockly.Ruby.ORDER_NONE) || '0';
         const branch = Blockly.Ruby.statementToCode(block, 'SUBSTACK') || '\n';
 
         return `${times}.times do\n${branch}end\n`;
```

Comment:
```suggestion
        return `repeat(${times}) do\n${branch}end\n`;
```

smalruby-libでは、loopでしたが、以下を実現するメソッドrepeatを用意する予定です。

```ruby
def repeat(times, &block)
  times.times do
    yield
    await # <- 1フレームだけ待つ
  end
end
```